src/dex-ui/pages/dao/ProposalCard.tsx

```diff
@@ -49,8 +52,16 @@ export const ProposalCard = (props: ProposalCardProps) => {
     </Flex>
   );
 
+  const goToProposalDetails = useCallback(() => {
```

Comment:
Any reason `useCallback` is being used here? I don't believe memoization is needed for this function.

Reply:
You are right. removed the callback hook.